build_tools/third_party/mlir-hlo/CMakeLists.txt

```diff
@@ -15,18 +15,9 @@ external_cc_library(
   PACKAGE
     tensorflow
   NAME
-    mlir_hlo
+    external_mhlo_includes
```

Comment:
Wait was this the only reason we still needed this shim? :laughing:

Reply:
Sure was.